mock/docs/site-defaults.cfg

```diff
@@ -231,6 +231,10 @@
 # config_opts['plugin_conf']['procenv_enable'] = False
 # config_opts['plugin_conf']['procenv_opts'] = {}
 #
+# config_opts['plugin_conf']['showrc'] = True
```

Comment:
Do we really want to have this enabled by default for everybody?

Reply:
These logs would increase reproducibility and provide more insight into the build root.  I'm inclined to have it on by default for folks.

Reply:
I'm afraid there's unnecessary risk that we'll break something (production Copr/Koji etc.).  There were dozens of similar problems when we were executing rpm inside the chroot from host.  That said, I'd prefer to enable it by default a bit later.

Reply:
I'll update the diff to disable by default, squash, and have the PR updated shortly.